### PR TITLE
TABL ROWORCOLST serialize: Delete value C #1722

### PR DESCRIPTION
**Pull request by JohannesKonings**
https://github.com/larshp/abapGit/issues/1722

### CODE REVIEW

**Comment by larshp** [comment]
Any testing performed for this change?

**Comment by JohannesKonings** [comment]
On NW 7.52 and NW 7.31 no diffs displayed for TABL in repo https://github.com/Microsoft/ABAP-SDK-for-Azure.
With repo https://github.com/abapGit-tests/TABL_with_index no diffs for TABL on NW 7.52. Unfortunately diffs are display on NW 7.31.

image

**Comment by larshp** [comment]
repo https://github.com/abapGit-tests/TABL_with_index should be updated after merging this change

File: src/objects/zcl_abapgit_object_tabl.clas.abap [hunk 1]
```diff
@@ -410,6 +411,14 @@ CLASS ZCL_ABAPGIT_OBJECT_TABL IMPLEMENTATION.
            ls_dd09l-as4date,
            ls_dd09l-as4time.
 
+    ASSIGN COMPONENT 'ROWORCOLST' OF STRUCTURE ls_dd09l TO <lg_roworcolst>.
+    IF sy-subrc = 0.
+      IF <lg_roworcolst> = 'C'.
```
Comment (larshp) [inline src/objects/zcl_abapgit_object_tabl.clas.abap]: IF in IF, can be changed to `IF condition AND condition`

**Pull request merged**